Tests/scripts/validate_files_structure.py

```diff
@@ -178,12 +177,29 @@ def validate_schema(file_path, matching_regex=None):
 def changed_id(file_path):
     change_string = run_git_command("git diff HEAD {0}".format(file_path))
     if re.search("\+id: .*", change_string) or re.search("\-id: .*", change_string):
-        print_error("You've changed the ID of the playbook {0} please undo.".format(file_path))
+        print_error("You've changed the ID of the file {0} please undo.".format(file_path))
         return True
 
     return False
 
 
+def changed_integration_id(file_path):
```

Comment:
why do you need another command? just set the space as optional in the regex:
`'[+-](  )?id: .*'`